lib/layout/box_layout.dart

```diff
@@ -202,8 +202,8 @@ class BoxLayoutState extends WidgetState<BoxLayout> with TemplatedWidgetState {
       List<Widget> updatedChildren = [];
       for (var i=0; i<children.length; i++) {
         updatedChildren.add(children[i]);
-        final child = children[i] as BoxLayout;
-        if (i != children.length - 1 && child.controller.visible) {
+        final visibleChild = children[i] is HasController && ((children[i] as HasController).controller as WidgetController).visible;
```

Comment:
`children[i] is HasController && ((children[i] as HasController).controller is WidgetController)`

It returns a `bool` instead of a `WidgetController` to access visible bool property